### PR TITLE
Map the remaining inner classes

### PR DESCRIPTION
**Pull request by modmuss50**
Im not too sure with the BookEditScreen names, so feel free to provide better ideas.

There are 2 remaining unmapped classes that are covered by https://github.com/FabricMC/yarn/pull/1431

### CODE REVIEW

File: mappings/net/minecraft/client/gui/widget/ButtonWidget.mapping [hunk 1]
```diff
@@ -10,3 +10,4 @@ CLASS net/minecraft/class_4185 net/minecraft/client/gui/widget/ButtonWidget
 	CLASS class_4241 PressAction
 		METHOD onPress (Lnet/minecraft/class_4185;)V
 			ARG 1 button
+	CLASS class_5316 TooltipAction
```
Comment (shedaniel) [inline mappings/net/minecraft/client/gui/widget/ButtonWidget.mapping]: not really an action, maybe `TooltipSupplier`

**Review by shedaniel — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/gui/widget/ButtonWidget.mapping (hunk 1 above)
Comment (modmuss50) [reply mappings/net/minecraft/client/gui/widget/ButtonWidget.mapping]: Yeah, the reason I went for this is the method name is `onTooltip`. I think that will make more sense tho

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/client/gui/screen/ingame/BookEditScreen.mapping [hunk 2]
```diff
@@ -38,6 +39,30 @@ CLASS net/minecraft/class_473 net/minecraft/client/gui/screen/ingame/BookEditScr
...
+		FIELD field_24278 style Lnet/minecraft/class_2583;
+		FIELD field_24279 content Ljava/lang/String;
+		FIELD field_24280 text Lnet/minecraft/class_2561;
 		FIELD field_2853 y I
 		FIELD field_2854 x I
+		METHOD <init> (Lnet/minecraft/class_2583;Ljava/lang/String;II)V
+			ARG 2 content
+			ARG 3 x
+			ARG 4 y
+	CLASS class_5233 RenderCache
```
Comment (shedaniel) [inline mappings/net/minecraft/client/gui/screen/ingame/BookEditScreen.mapping]: is it really a cache, or it is just the current content, as this is an editing screen it is quite weird to have this name

Comment (modmuss50) [reply mappings/net/minecraft/client/gui/screen/ingame/BookEditScreen.mapping]: PageContent? might be a better name? I struggled with this one for a bit.

Comment (liach) [reply mappings/net/minecraft/client/gui/screen/ingame/BookEditScreen.mapping]: or `PageContent`? should mention page if this is page specific.

Comment (shedaniel) [reply mappings/net/minecraft/client/gui/screen/ingame/BookEditScreen.mapping]: PageContent should work

Comment (modmuss50) [reply mappings/net/minecraft/client/gui/screen/ingame/BookEditScreen.mapping]: Now uses PageContent

**Review by liach — APPROVED** [review]

**Review by dblsaiko — APPROVED** [review]

File: mappings/net/minecraft/client/gui/screen/ingame/BookEditScreen.mapping [hunk 3]
```diff
@@ -38,6 +39,30 @@ CLASS net/minecraft/class_473 net/minecraft/client/gui/screen/ingame/BookEditScr
...
+		FIELD field_24273 position Lnet/minecraft/class_473$class_5234;
+		FIELD field_24276 lines [Lnet/minecraft/class_473$class_475;
+		METHOD <init> (Ljava/lang/String;Lnet/minecraft/class_473$class_5234;Z[I[Lnet/minecraft/class_473$class_475;[Lnet/minecraft/class_768;)V
+			ARG 1 pageContent
+			ARG 5 lines
+	CLASS class_5234 Position
+		FIELD field_24281 x I
+		FIELD field_24282 y I
+		METHOD <init> (II)V
+			ARG 1 z
```
Comment (YanisBft) [inline mappings/net/minecraft/client/gui/screen/ingame/BookEditScreen.mapping]:
```suggestion
			ARG 1 x
```

**Review by YanisBft — APPROVED** [review]

**Pull request merged**